client/package.json

```diff
@@ -3,9 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
+    "@mui/icons-material": "^5.0.0",
+    "@mui/lab": "*",
+    "@mui/material": "^5.0.0",
```

Comment:
I think it's a good idea to upgrade `mui v5`, although keeping both versions might be undesired. Just something to keep in mind, I don't think we need to worry about this too much at the moment.

Reply:
Yes, I noticed that `mui v5` came out recently. (Maybe after we started the project?)
I am not 100% sure, but I guess some components work a bit differently with Typescript. (Or, it works, but Typescript complains due to type?)

(e.g., `Grid from '@mui/material/Grid'` and `Grid from '@material-ui/core/Grid'`)
I had some issues when I was trying to use the `spacing` prop like `spacing={{ xs: 0, sm: 1, md: 3, lg: 5, xl: 10 }}` with `@material-ui/core/Grid`. I guess `object` value is not supported in `@material-ui/core/Grid`